### PR TITLE
update IProfile

### PR DESCRIPTION
**Pull request by adrien-nf**

### CODE REVIEW

File: server/interfaces/IProfile.ts [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+    Happy,
+    Sad,
+    Angry,
+    Surprised,
+    Neutral,
+    Cyclops,
+    Astonished,
+    High,
+    Meh,
+    Triclops,
```
Comment (cyrilgourgouillon) [inline server/interfaces/IProfile.ts:36]: Nice names 👍

File: server/interfaces/IProfile.ts [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
     username: string,
-    hat: number,
-    body: number,
-    lead: number,
+    rubberColor: RubberColor,
+    bodyColor: BodyColor,
+    faceType: FaceType,
+}
+
+export enum RubberColor {
```
Comment (cyrilgourgouillon) [inline server/interfaces/IProfile.ts:8]: I've seen somewhere that Enums should be in capital letters ?
Is this still relevant @adrien-nf ?

File: server/interfaces/IProfile.ts (hunk 1 above)
Comment (adrien-nf) [reply server/interfaces/IProfile.ts:36]: Thanks! 👍

File: server/interfaces/IProfile.ts (hunk 2 above)
Comment (adrien-nf) [reply server/interfaces/IProfile.ts:8]: So after a bit of searching, I've found that the Enum being all-capital hail from Java language:
```java
enum Level {
  LOW,
  MEDIUM,
  HIGH
}
```
Source: [https://www.w3schools.com/java/java_enums.asp](https://www.w3schools.com/java/java_enums.asp)

As it turns out, typescript convention differ in naming the enum members:

> Use PascalCase for enum member
> > Reason: Convention followed by TypeScript team i.e. the language creators e.g ```SyntaxKind.StringLiteral```. Also helps with translation (code generation) of other languages into TypeScript.
```ts
enum Color {
    Red
}
```
Source: [https://basarat.gitbook.io/typescript/styleguide#enum](https://basarat.gitbook.io/typescript/styleguide#enum)

So I figured, if TypeScript team uses PascalCase Enums, we might as well do so too. **What do you think @cyrilgourgouillon ?**

Nota bene: I would, however, use CAPS_SNAKE_CASE for class constants.

Comment (cyrilgourgouillon) [reply server/interfaces/IProfile.ts:8]: Ok !

**Review by cyrilgourgouillon — APPROVED** [review]

**Pull request merged**